druid-shell/src/keyboard.rs

```diff
@@ -271,4 +582,26 @@ mod tests {
         assert!(!s_16.is_char_boundary(15));
         let _too_big = TinyStr::new("😍🥰😘😗");
     }
+
+    #[test]
+    fn key_matching() {
+        use super::Modifier::*;
+        let key = KeyEvent::for_test(NoMods, "a", KeyCode::KeyA);
+        assert!(key.matches('a'));
```

Comment:
following macOS, we allow a key's text equivalent to be multiple chars. I don't know of an actual keyboard layout where this is exercised, but I could certainly imagine it with indic languages?

Reply:
At that point, we shouldn't be using key events but IME.